k8s/mongodb/mongo-pv.yaml

```diff
@@ -0,0 +1,41 @@
+#############################################################
+# This YAML section desribes a k8s PV for mongodb dbPath    #
+#############################################################
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: pvc-24cdc52d-aea2-11e7-9633-000d3a2b8d7c
```

Comment:
Where did this value (pvc-24cdc52d-aea2-11e7-9633-000d3a2b8d7c) come from? It seems awfully specific.

Reply:
this should be configurable will change :)